src/app/external-cluster-wizard/steps/external-cluster/provider/eks/template.html

```diff
@@ -52,34 +52,54 @@
       </mat-error>
     </mat-form-field>
 
-    <mat-form-field *ngIf="isDialogView()">
-      <mat-label>Kubernetes Version</mat-label>
-      <input [formControlName]="Controls.Version"
-             matInput
-             type="text"
-             autocomplete="off" />
-      <mat-hint>Selected kubernetes version for your nodes.</mat-hint>
-    </mat-form-field>
-
-    <km-autocomplete *ngIf="!isDialogView()"
+    <km-autocomplete *ngIf="!isDialogView(); else dialogViewKubernetesVersion"
                      label="Kubernetes Version"
                      required="true"
                      [options]="kubernetesVersions"
                      [formControlName]="Controls.Version">
       <ng-container hint>
-        Set the kubernetes version.
+        Enter or Select Kubernetes Version
       </ng-container>
     </km-autocomplete>
 
+    <ng-template #dialogViewKubernetesVersion>
+      <mat-form-field *ngIf="isDialogView()">
```

Comment:
do we need this condition here since we using `else` in line 55

Reply:
hmm may be correct